modules/cms/classes/layout/fields.yaml

```diff
@@ -32,6 +32,13 @@ secondaryTabs:
             type: codeeditor
             language: twig
 
+        notice:
+            tab: cms::lang.editor.code
+            type: partial
+            hidden: true
+            path: ~/modules/cms/controllers/index/_notice_field.htm
```

Comment:
You should be able to change the field name to safemode_notice and rename the partial `_safemode_notice.htm` and then remove the path property from all these configs. What is the p-b-0 class for?